scripts/importer/validateData.py

```diff
@@ -5644,7 +5667,10 @@ def main_validate(args):
     # set portal version
     cbio_version = portal_instance.portal_version
 
-    validate_study(study_dir, portal_instance, logger, relaxed_mode, strict_maf_checks)
+    if partial_data:
+        validate_data_dir(data_dir, portal_instance, logger, relaxed_mode, strict_maf_checks)
+    else:
+        validate_study(data_dir, portal_instance, logger, relaxed_mode, strict_maf_checks)
```

Comment:
Maybe this addresses my concern above about validating "whole study import" ... there is a separate method here, so maybe the validation approach for whole study import is different.